bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/gnirs/GnirsRule.java

```diff
@@ -43,6 +43,33 @@ public IConfigMatcher getMatcher() {
         }
     };
 
+    private static final IConfigRule SHORT_RED_CAMERA_RULE = new IConfigRule() {
+        private static final String MESSAGE = "The short red camera is currently unavailable.";
+
+        @Override
+        public Problem check(final Config config, final int step, final ObservationElements elems, final Object state) {
+            final Camera camera = (Camera) SequenceRule.getInstrumentItem(config, InstGNIRS.CAMERA_PROP);
+            if (camera == null) return null;
+            switch (camera) {
+                case SHORT_BLUE:
+                case LONG_BLUE:
+                    return null;
+            }
```

Comment:
There's also a `LONG_RED`, wouldn't that go in the list above?

Reply:
I think you're right though we probably can remove this whole check altogether